bot/exts/utils/snekbox.py

```diff
@@ -222,14 +290,17 @@ async def send_eval(self, ctx: Context, code: str) -> Message:
                 response = await ctx.send(msg, allowed_mentions=allowed_mentions)
             scheduling.create_task(wait_for_deletion(response, (ctx.author.id,)), event_loop=self.bot.loop)
 
-            log.info(f"{ctx.author}'s job had a return code of {results['returncode']}")
+            log.info(f"{ctx.author}'s {job_name} job had a return code of {results['returncode']}")
         return response
 
-    async def continue_eval(self, ctx: Context, response: Message) -> Optional[str]:
+    async def continue_eval(
+        self, ctx: Context, response: Message, command: Command
+    ) -> tuple[Optional[str], Optional[list[str]]]:
         """
         Check if the eval session should continue.
 
-        Return the new code to evaluate or None if the eval session should be terminated.
+        If the code is to be evaluated, return the new code and the args if the command is the timeit command.
+        Otherwise return None if the eval session should be terminated.
```

Comment:
```suggestion
        If the code is to be re-evaluated, return the new code, and the args if the command is the timeit command.
        Otherwise return None as the first element if the eval session should be terminated.
```

It took me a bit to understand these two sentences, I believe the above suggestion makes it a bit better but maybe it could use a full reword. The code also never returns None alone so the second sentence should mention it'll return None for the code in the tuple if it's terminated